packages/shared/__tests__/helpers/boot.tsx

```diff
@@ -35,18 +36,35 @@ interface TestBootProviderProps {
   gb?: GrowthBook;
 }
 
-export const settingsContext: Partial<SettingsContextData> = {
-  spaciness: 'eco',
+export const settingsContext: SettingsContextData = {
```

Comment:
One day we should really make a neat settings wrapper else everytime there's something new we need to add it in 100 tests